chains/botanists.json

```diff
@@ -0,0 +1,417 @@
+      },
+      "choices": [
+        {
+          "text": "“So what? The ecosystems won't be stable until the last species has been completely integrated. This kind of things happen all the time.”",
+          "next": [
+            {
+              "event": "ignore",
+              "in": 2
+            }
+          ]
+        },
+        {
+          "text": "“Someone should probably go there and see if there's anything weird going on.”",
+          "next": [
+            {
+              "event": "send_intern_to_investigate"
+            }
+          ]
+        },
+        {
+          "text": "“Is that normal? Maybe I should ask AUDE to run some simulations about it.”",
+          "next": [
+            {
+              "event": "aude_simulation"
+            }
+          ]
+        }
+      ]
+    },
+    "send_intern_to_investigate": {
+      "text": "Turns out the only member of your team available is an intern. It takes some time explaining the situation to them and what you're expecting them to do, but they eventually seem to understand.",
+      "next": [
+        {
+          "event": "news_from_intern",
+          "in": 1
+        }
+      ]
+    },
+    "news_from_intern": {
+      "text": "You get news from the intern you sent to investigate about the forests that turned to plains and deserts in the northern hemisphere.",
+      "next": [
+        {
+          "event": "intern_hurt"
+        },
+        {
+          "event": "intern_misunderstood",
+          "weight": 2
+        }
+      ]
+    },
+    "intern_hurt": {
+      "text": "They've managed to hurt themselves while on the field before they found any insight about what's going on there. Healing them will cost around 50€€€.",
+      "effects": {
+        "pay_to_heal_intern": true
+      },
+      "next": [
+        {
+          "event": "space_botanists_call",
+          "in": 1
+        }
+      ]
+    },
+    "intern_misunderstood": {
+      "text": "They proudly tell you that yes, they are positive there are no forests there. When asked about the reasons why there are no forests anymore, they seem lost.",
+      "choices": [
+        {
+          "text": "“I guess I should have repeated my instructions an 11th time...”",
+          "next": [
+            {
+              "event": "space_botanists_call",
+              "in": 1
+            }
+          ]
+        }
+      ]
+    },
+    "aude_simulation": {
+      "text": "AUDE informs you running the simulations, even based on cached data rather than field observations, will take some time.",
+      "next": [
+        {
+          "event": "aude_simulation_completed",
+          "in": 1
+        }
+      ]
+    },
+    "aude_simulation_completed": {
+      "text": "AUDE lets you know it's done running the simulations about the forests that turned to plains and deserts in the northern hemisphere.\nAccording to its conclusions, this is but a temporary anomaly, and the plants and trees are bound to grow back there soon.",
```

Comment:
j'avais plus vu AUDE comme unE IA

Reply:
je trouve que la désigner ainsi la rend + humaine ce que je préfère narrativement

Reply:
pas sûr de comprendre l'incohérence entre ce que j'ai écrit et ce que tu dis

Reply:
she's done vs it's done

Reply:
bah pour moi IA sous-entendait pas forcément genrée